development/sass/modules/_hover-card.scss

```diff
@@ -0,0 +1,115 @@
+.blog-preview-list {
+  padding-right: 300px;
+  position: relative;
+
+  @media (max-width: 572px) {
+    padding: 0;
+  }
+
+  ul {
+    width:100%;
+    padding-top: 16px;
+
+    @media (max-width: 572px) {
+      padding: 0;
+    }
+  }
+
+  li {
+    width:100%;
+    padding: 0 16px;
+
+    @media (max-width: 572px) {
+      padding: 0;
+    }
+  }
+}
+
+
+.hover-card {
+  display: block;
+  width: 100%;
+  min-height: 192px;
+  padding-left: 192px;
+  position:relative;
+  border: 2px solid rgba(0,0,0,0);
+  max-width: 700px;
+  min-width: 540px;
+  transition: box-shadow 0.25s, border 0.25s, border-radius 0.25s;
+
+  @media (max-width: 572px) {
```

Comment:
I'm going to merge this for now but I highly reccomend creating a variable for the mobile width and storing it in a `_settings.scss` file so you can change it in one place an update the entire site. Ex. 

```
$mobile-width: 572px; 

@media (max-width: $mobile-width) {
... 
```